matlab/test/arrow/buffer/tBuffer.m

```diff
@@ -69,7 +69,7 @@ function fromMATLABFromEmptyUint8(testCase)
 
         function fromMATLABUint8Vector(testCase)
```

Comment:
Sorry, I think I meant to imply with my previous comment that we should capitalize all of the `Uint` to `UInt` for consistency.